ui/desktop/app/routes/scopes/scope/projects/sessions/index.js

```diff
@@ -113,6 +123,7 @@ export default class ScopesScopeProjectsSessionsIndexRoute extends Route {
       allSessions: this.allSessions,
       allTargets: this.allTargets,
       totalItems,
+      isClientDaemonRunning: await this.ipc.invoke('isClientDaemonRunning'),
```

Comment:
It's so annoying as I didn't realize this but I originally wanted this to be inside a helper but apparently ember can't handle promises inside templates without me using a library like [this](https://github.com/fivetanley/ember-promise-helpers) which I didn't want to pull in. 

I just added this to the model instead and we can remove it in the future.